addons/l10n_ar/models/account_fiscal_position.py

```diff
@@ -19,6 +19,5 @@ def _get_fiscal_position(self, partner, delivery=None):
 
     def _prepare_fpos_base_domain(self, vat_required):
         domain = super()._prepare_fpos_base_domain(vat_required)
-        if self._context.get('l10n_ar_afip_responsibility_type_id'):
```

Comment:
I would do: 
if 'l10n_ar_afip_responsibility_type_id in self._context.keys():

That way we avoid the extra term in other non-AR companies in the db. 

@pablohmontenegro

Reply:
Indeed. It's better. :+1: